datafusion/common/src/config.rs

```diff
@@ -1653,10 +1649,6 @@ config_namespace_with_hashmap! {
         /// Sets bloom filter number of distinct values. If NULL, uses
         /// default parquet options
         pub bloom_filter_ndv: Option<u64>, default = None
-
-        /// Sets max statistics size for the column path. If NULL, uses
-        /// default parquet options
-        pub max_statistics_size: Option<usize>, default = None
```

Comment:
I think it is a mistake to remove this field directly -- we should put it back in and figure out some way to mark it deprecated